pkg/gitops/flux/flux.go

```diff
@@ -149,6 +151,78 @@ func (f *Flux) Uninstall(ctx context.Context, cluster *types.Cluster, clusterSpe
 	return nil
 }
 
+func (f *Flux) PauseClusterResourcesReconcile(ctx context.Context, cluster *types.Cluster, clusterSpec *cluster.Spec, provider providers.Provider) error {
+	if f.shouldSkipFlux() {
+		logger.Info("GitOps field not specified, pause cluster resources reconcile skipped")
+		return nil
+	}
+
+	c, err := f.fluxClient.GetCluster(ctx, cluster, clusterSpec)
```

Comment:
can we just pass the currentSpec to this method? And avoid an extra get

Reply:
yea actually we don't need this check